recipe/meta.yaml

```
@@ -17,9 +17,9 @@ requirements:
     - {{ compiler('c') }}
   host:
     - python
-    - cython 0.29
-    - libxml2 2.10
-    - libxslt 1.1.37
+    - cython 3.0.10
```

```suggestion
    - cython >=3.0.10
```
cython usually doesn't need a strict pinning. And a strict pinning may render further updates more difficult.